[2023-04-16T02:18:55.351+0000] {processor.py:153} INFO - Started process (PID=246) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:18:55.354+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:18:55.359+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:18:55.358+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:18:55.405+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:18:55.395+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:18:55.408+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:18:55.462+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.118 seconds
[2023-04-16T02:19:26.305+0000] {processor.py:153} INFO - Started process (PID=273) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:19:26.310+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:19:26.314+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:19:26.314+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:19:26.386+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:19:26.366+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:19:26.390+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:19:26.663+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.368 seconds
[2023-04-16T02:19:57.510+0000] {processor.py:153} INFO - Started process (PID=298) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:19:57.523+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:19:57.528+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:19:57.527+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:19:57.567+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:19:57.561+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:19:57.570+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:19:57.724+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.223 seconds
[2023-04-16T02:20:28.573+0000] {processor.py:153} INFO - Started process (PID=327) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:20:28.577+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:20:28.591+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:20:28.591+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:20:28.653+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:20:28.641+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:20:28.705+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:20:28.916+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.365 seconds
[2023-04-16T02:20:59.212+0000] {processor.py:153} INFO - Started process (PID=354) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:20:59.216+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:20:59.221+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:20:59.221+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:20:59.256+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:20:59.251+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:20:59.260+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:20:59.302+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.102 seconds
[2023-04-16T02:21:29.988+0000] {processor.py:153} INFO - Started process (PID=381) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:21:29.992+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:21:29.997+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:21:29.997+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:21:30.045+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:21:30.037+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:21:30.048+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:21:30.111+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.132 seconds
[2023-04-16T02:22:01.033+0000] {processor.py:153} INFO - Started process (PID=407) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:22:01.038+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:22:01.043+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:22:01.043+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:22:01.086+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:22:01.080+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:22:01.093+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:22:01.170+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.144 seconds
[2023-04-16T02:22:31.787+0000] {processor.py:153} INFO - Started process (PID=435) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:22:31.793+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:22:31.799+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:22:31.798+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:22:31.830+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:22:31.825+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:22:31.833+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:22:31.890+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.115 seconds
[2023-04-16T02:23:03.028+0000] {processor.py:153} INFO - Started process (PID=461) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:23:03.035+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:23:03.040+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:23:03.039+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:23:03.116+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:23:03.098+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:23:03.126+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:23:03.279+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.286 seconds
[2023-04-16T02:23:34.000+0000] {processor.py:153} INFO - Started process (PID=489) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:23:34.010+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:23:34.014+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:23:34.014+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:23:34.081+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:23:34.072+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:23:34.085+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:23:34.205+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.233 seconds
[2023-04-16T02:24:04.883+0000] {processor.py:153} INFO - Started process (PID=518) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:24:04.887+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:24:04.891+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:24:04.890+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:24:04.924+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:24:04.914+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:24:04.929+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:24:05.003+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.132 seconds
[2023-04-16T02:24:35.696+0000] {processor.py:153} INFO - Started process (PID=545) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:24:35.701+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:24:35.706+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:24:35.705+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:24:35.735+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:24:35.730+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:24:35.737+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:24:35.781+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.093 seconds
[2023-04-16T02:25:06.389+0000] {processor.py:153} INFO - Started process (PID=572) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:25:06.404+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:25:06.409+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:25:06.408+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:25:06.439+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:25:06.434+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:25:06.444+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:25:06.484+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.102 seconds
[2023-04-16T02:25:37.205+0000] {processor.py:153} INFO - Started process (PID=599) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:25:37.208+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:25:37.212+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:25:37.212+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:25:37.239+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:25:37.234+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:25:37.241+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:25:37.287+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.089 seconds
[2023-04-16T02:26:07.932+0000] {processor.py:153} INFO - Started process (PID=626) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:26:07.936+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:26:07.941+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:26:07.940+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:26:07.973+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:26:07.965+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:26:07.977+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:26:08.070+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.143 seconds
[2023-04-16T02:26:38.673+0000] {processor.py:153} INFO - Started process (PID=660) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:26:38.690+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:26:38.704+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:26:38.702+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:26:38.783+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:26:38.773+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:26:38.789+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:26:38.865+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.202 seconds
[2023-04-16T02:27:09.469+0000] {processor.py:153} INFO - Started process (PID=688) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:27:09.476+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:27:09.482+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:27:09.482+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:27:09.521+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:27:09.517+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:27:09.523+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:27:09.579+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.121 seconds
[2023-04-16T02:27:40.415+0000] {processor.py:153} INFO - Started process (PID=714) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:27:40.421+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:27:40.427+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:27:40.426+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:27:40.471+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:27:40.463+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:27:40.486+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:27:40.546+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.139 seconds
[2023-04-16T02:28:11.248+0000] {processor.py:153} INFO - Started process (PID=741) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:28:11.254+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:28:11.258+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:28:11.257+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:28:11.296+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:28:11.291+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:28:11.301+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:28:11.351+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.111 seconds
[2023-04-16T02:28:42.315+0000] {processor.py:153} INFO - Started process (PID=769) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:28:42.324+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:28:42.331+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:28:42.331+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:28:42.367+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:28:42.361+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:28:42.371+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:28:42.464+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.158 seconds
[2023-04-16T02:29:13.309+0000] {processor.py:153} INFO - Started process (PID=797) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:29:13.315+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:29:13.319+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:29:13.319+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:29:13.352+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:29:13.345+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:29:13.356+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:29:13.426+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.128 seconds
[2023-04-16T02:29:44.126+0000] {processor.py:153} INFO - Started process (PID=824) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:29:44.131+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:29:44.134+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:29:44.133+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:29:44.166+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:29:44.159+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:29:44.172+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:29:44.226+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.115 seconds
[2023-04-16T02:30:14.820+0000] {processor.py:153} INFO - Started process (PID=851) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:30:14.825+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:30:14.837+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:30:14.836+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:30:14.886+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:30:14.879+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:30:14.888+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:30:14.931+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.125 seconds
[2023-04-16T02:30:45.427+0000] {processor.py:153} INFO - Started process (PID=878) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:30:45.430+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:30:45.434+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:30:45.433+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:30:45.457+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:30:45.453+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:30:45.462+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:30:45.506+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.085 seconds
[2023-04-16T02:31:16.007+0000] {processor.py:153} INFO - Started process (PID=905) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:31:16.010+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:31:16.014+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:31:16.013+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:31:16.043+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:31:16.038+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:31:16.046+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:31:16.094+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.098 seconds
[2023-04-16T02:31:46.594+0000] {processor.py:153} INFO - Started process (PID=931) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:31:46.599+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:31:46.604+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:31:46.603+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:31:46.633+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:31:46.627+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:31:46.636+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:31:46.713+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.126 seconds
[2023-04-16T02:32:17.212+0000] {processor.py:153} INFO - Started process (PID=959) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:32:17.218+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:32:17.221+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:32:17.221+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:32:17.243+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:32:17.240+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:32:17.245+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:32:17.287+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.083 seconds
[2023-04-16T02:32:47.779+0000] {processor.py:153} INFO - Started process (PID=984) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:32:47.783+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:32:47.789+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:32:47.788+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:32:47.848+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:32:47.837+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:32:47.852+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:32:47.919+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.156 seconds
[2023-04-16T02:33:18.430+0000] {processor.py:153} INFO - Started process (PID=1014) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:33:18.434+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:33:18.438+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:33:18.437+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:33:18.476+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:33:18.473+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:33:18.480+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:33:18.556+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.140 seconds
[2023-04-16T02:33:49.081+0000] {processor.py:153} INFO - Started process (PID=1041) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:33:49.085+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:33:49.091+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:33:49.090+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:33:49.119+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:33:49.115+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:33:49.121+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:33:49.157+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.088 seconds
[2023-04-16T02:34:19.810+0000] {processor.py:153} INFO - Started process (PID=1069) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:34:19.814+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T02:34:19.817+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:34:19.816+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:34:19.841+0000] {logging_mixin.py:137} INFO - [2023-04-16T02:34:19.837+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T02:34:19.845+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T02:34:19.885+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.082 seconds
[2023-04-16T04:07:52.194+0000] {processor.py:153} INFO - Started process (PID=181) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:07:52.211+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:07:52.216+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:07:52.215+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:07:52.271+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:07:52.266+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:07:52.273+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:07:52.323+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.144 seconds
[2023-04-16T04:08:23.155+0000] {processor.py:153} INFO - Started process (PID=206) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:08:23.159+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:08:23.162+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:08:23.162+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:08:23.189+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:08:23.185+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:08:23.194+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:08:23.285+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.140 seconds
[2023-04-16T04:08:53.901+0000] {processor.py:153} INFO - Started process (PID=232) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:08:53.906+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:08:53.910+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:08:53.910+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:08:53.944+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:08:53.938+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:08:53.951+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:08:54.014+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.121 seconds
[2023-04-16T04:09:24.549+0000] {processor.py:153} INFO - Started process (PID=259) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:09:24.555+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:09:24.557+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:09:24.557+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:09:24.585+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:09:24.581+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:09:24.595+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:09:24.638+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.096 seconds
[2023-04-16T04:09:55.178+0000] {processor.py:153} INFO - Started process (PID=285) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:09:55.183+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:09:55.190+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:09:55.189+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:09:55.229+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:09:55.224+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:09:55.233+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:09:55.293+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.130 seconds
[2023-04-16T04:10:25.757+0000] {processor.py:153} INFO - Started process (PID=313) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:10:25.760+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:10:25.763+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:10:25.763+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:10:25.793+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:10:25.788+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:10:25.796+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:10:25.847+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.104 seconds
[2023-04-16T04:10:56.325+0000] {processor.py:153} INFO - Started process (PID=340) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:10:56.327+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:10:56.332+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:10:56.332+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:10:56.376+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:10:56.371+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:10:56.378+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:10:56.434+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.119 seconds
[2023-04-16T04:11:26.773+0000] {processor.py:153} INFO - Started process (PID=368) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:11:26.777+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:11:26.781+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:11:26.781+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:11:26.812+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:11:26.808+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:11:26.815+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:11:27.097+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.329 seconds
[2023-04-16T04:11:57.642+0000] {processor.py:153} INFO - Started process (PID=401) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:11:57.647+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:11:57.651+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:11:57.650+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:11:57.693+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:11:57.684+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:11:57.698+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:11:57.787+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.156 seconds
[2023-04-16T04:12:28.322+0000] {processor.py:153} INFO - Started process (PID=430) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:12:28.335+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:12:28.339+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:12:28.339+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:12:28.382+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:12:28.376+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:12:28.388+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:12:28.488+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.180 seconds
[2023-04-16T04:12:59.202+0000] {processor.py:153} INFO - Started process (PID=456) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:12:59.218+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:12:59.228+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:12:59.223+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:12:59.371+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:12:59.318+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:12:59.387+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:12:59.490+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.314 seconds
[2023-04-16T04:13:30.062+0000] {processor.py:153} INFO - Started process (PID=485) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:13:30.068+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:13:30.072+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:13:30.071+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:13:30.095+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:13:30.091+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:13:30.096+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:13:30.140+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.085 seconds
[2023-04-16T04:14:00.572+0000] {processor.py:153} INFO - Started process (PID=510) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:14:00.576+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:14:00.579+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:14:00.579+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:14:00.608+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:14:00.601+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:14:00.612+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:14:00.658+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.097 seconds
[2023-04-16T04:14:31.150+0000] {processor.py:153} INFO - Started process (PID=539) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:14:31.153+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:14:31.156+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:14:31.155+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:14:31.176+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:14:31.172+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:14:31.179+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:14:31.216+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.072 seconds
[2023-04-16T04:15:01.607+0000] {processor.py:153} INFO - Started process (PID=564) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:15:01.610+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:15:01.613+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:15:01.613+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:15:01.643+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:15:01.638+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:15:01.646+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:15:01.686+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.088 seconds
[2023-04-16T04:15:32.189+0000] {processor.py:153} INFO - Started process (PID=594) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:15:32.193+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:15:32.196+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:15:32.196+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:15:32.223+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:15:32.217+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:15:32.228+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:15:32.273+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.091 seconds
[2023-04-16T04:16:02.780+0000] {processor.py:153} INFO - Started process (PID=619) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:16:02.786+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:16:02.790+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:16:02.790+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:16:02.840+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:16:02.834+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:16:02.842+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:16:02.905+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.139 seconds
[2023-04-16T04:16:33.442+0000] {processor.py:153} INFO - Started process (PID=648) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:16:33.446+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:16:33.449+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:16:33.448+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:16:33.471+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:16:33.467+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:16:33.473+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:16:33.511+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.075 seconds
[2023-04-16T04:17:04.111+0000] {processor.py:153} INFO - Started process (PID=674) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:17:04.117+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:17:04.123+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:17:04.122+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:17:04.151+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:17:04.146+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:17:04.164+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:17:04.222+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.121 seconds
[2023-04-16T04:17:34.678+0000] {processor.py:153} INFO - Started process (PID=701) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:17:34.683+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:17:34.686+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:17:34.686+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:17:34.706+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:17:34.703+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:17:34.709+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:17:34.759+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.087 seconds
[2023-04-16T04:18:05.426+0000] {processor.py:153} INFO - Started process (PID=727) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:18:05.430+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:18:05.432+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:18:05.432+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:18:05.456+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:18:05.450+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:18:05.459+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:18:05.509+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.090 seconds
[2023-04-16T04:18:35.857+0000] {processor.py:153} INFO - Started process (PID=752) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:18:35.860+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:18:35.866+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:18:35.865+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:18:35.893+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:18:35.886+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:18:35.897+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:18:35.975+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.125 seconds
[2023-04-16T04:19:06.966+0000] {processor.py:153} INFO - Started process (PID=780) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:19:06.972+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:19:06.998+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:19:06.998+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:19:07.492+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:19:07.465+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:19:07.499+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:19:08.466+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 1.506 seconds
[2023-04-16T04:19:39.327+0000] {processor.py:153} INFO - Started process (PID=807) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:19:39.342+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:19:39.355+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:19:39.355+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:19:39.424+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:19:39.418+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:19:39.446+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:19:39.524+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.213 seconds
[2023-04-16T04:20:09.904+0000] {processor.py:153} INFO - Started process (PID=833) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:20:09.909+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:20:09.912+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:20:09.911+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:20:09.932+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:20:09.928+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:20:09.936+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:20:09.990+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.092 seconds
[2023-04-16T04:20:40.560+0000] {processor.py:153} INFO - Started process (PID=860) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:20:40.565+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:20:40.568+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:20:40.568+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:20:40.596+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:20:40.591+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:20:40.598+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:20:40.643+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.090 seconds
[2023-04-16T04:21:11.214+0000] {processor.py:153} INFO - Started process (PID=887) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:21:11.221+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:21:11.228+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:21:11.228+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:21:11.267+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:21:11.262+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:21:11.271+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:21:11.350+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.146 seconds
[2023-04-16T04:21:41.743+0000] {processor.py:153} INFO - Started process (PID=914) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:21:41.750+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:21:41.753+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:21:41.753+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:21:41.777+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:21:41.773+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:21:41.780+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:21:41.831+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.098 seconds
[2023-04-16T04:22:12.144+0000] {processor.py:153} INFO - Started process (PID=941) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:22:12.146+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:22:12.150+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:22:12.149+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:22:12.170+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:22:12.166+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:22:12.172+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:22:12.213+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.075 seconds
[2023-04-16T04:22:42.978+0000] {processor.py:153} INFO - Started process (PID=969) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:22:42.990+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:22:43.008+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:22:43.007+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:22:43.078+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:22:43.067+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:22:43.097+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:22:43.182+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.213 seconds
[2023-04-16T04:23:13.759+0000] {processor.py:153} INFO - Started process (PID=995) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:23:13.763+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:23:13.766+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:23:13.765+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:23:13.818+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:23:13.804+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:23:13.829+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:23:13.885+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.137 seconds
[2023-04-16T04:23:44.216+0000] {processor.py:153} INFO - Started process (PID=1022) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:23:44.225+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:23:44.234+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:23:44.234+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:23:44.259+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:23:44.255+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:23:44.264+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:23:44.311+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.102 seconds
[2023-04-16T04:24:14.773+0000] {processor.py:153} INFO - Started process (PID=1049) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:24:14.804+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:24:14.819+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:24:14.819+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:24:14.852+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:24:14.839+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:24:14.859+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:24:14.926+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.164 seconds
[2023-04-16T04:24:45.193+0000] {processor.py:153} INFO - Started process (PID=1076) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:24:45.199+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:24:45.207+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:24:45.206+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:24:45.265+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:24:45.244+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:24:45.297+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:24:45.376+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.195 seconds
[2023-04-16T04:25:16.051+0000] {processor.py:153} INFO - Started process (PID=1103) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:25:16.057+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:25:16.062+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:25:16.062+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:25:16.095+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:25:16.089+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:25:16.099+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:25:16.146+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.104 seconds
[2023-04-16T04:25:46.841+0000] {processor.py:153} INFO - Started process (PID=1131) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:25:46.855+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:25:46.858+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:25:46.858+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:25:46.930+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:25:46.913+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:25:46.936+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:25:47.095+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.278 seconds
[2023-04-16T04:26:17.834+0000] {processor.py:153} INFO - Started process (PID=1157) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:26:17.838+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:26:17.842+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:26:17.842+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:26:17.868+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:26:17.864+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:26:17.872+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:26:17.923+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.095 seconds
[2023-04-16T04:26:48.489+0000] {processor.py:153} INFO - Started process (PID=1185) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:26:48.496+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:26:48.501+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:26:48.501+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:26:48.532+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:26:48.528+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:26:48.534+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:26:48.577+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.096 seconds
[2023-04-16T04:27:19.395+0000] {processor.py:153} INFO - Started process (PID=1212) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:27:19.398+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:27:19.401+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:27:19.401+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:27:19.423+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:27:19.419+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:27:19.425+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:27:19.462+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.073 seconds
[2023-04-16T04:27:50.221+0000] {processor.py:153} INFO - Started process (PID=1239) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:27:50.227+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:27:50.232+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:27:50.232+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:27:50.265+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:27:50.261+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:27:50.268+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:27:50.311+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.097 seconds
[2023-04-16T04:28:20.931+0000] {processor.py:153} INFO - Started process (PID=1266) to work on /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:28:20.941+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/dag_rocket.py for tasks to queue
[2023-04-16T04:28:20.949+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:28:20.949+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:28:20.979+0000] {logging_mixin.py:137} INFO - [2023-04-16T04:28:20.973+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/dag_rocket.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_rocket.py", line 4, in <module>
    from rockets import extract_images, extract_rockets
ModuleNotFoundError: No module named 'rockets'
[2023-04-16T04:28:20.981+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_rocket.py
[2023-04-16T04:28:21.023+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/dag_rocket.py took 0.110 seconds
